scripts/check_frontend_test_coverage.py

```diff
@@ -0,0 +1,217 @@
+
+
+def run_frontend_tests_script():
+    """Run the frontend tests script using subprocess."""
+    subprocess.check_call([
+        'python', '-m', 'scripts.run_frontend_tests'])
+
+
+def create_tmp_folder():
+    """Creates a temporary folder."""
+    if not os.path.exists('./tmp'):
+        os.mkdir('./tmp')
+
+
+def delete_tmp_folder():
+    """Delete the temporary folder."""
+    if os.path.exists('./tmp'):
+        shutil.rmtree('./tmp')
+
+
+def filter_lines(line):
+    """Check if the line has file path (SF) or total lines (LF) or covered
+        lines (LH) of the test.
+
+    Args:
+        line: str. A line from lcov file.
+
+    Returns:
+        Boolean. If the line has the file path or total lines or covered lines
+        of the test.
+    """
+    return (line.find('SF') >= 0 or line.find('LH') >= 0
+            or line.find('LF') >= 0)
+
+
+def get_test_file_name(test_path):
+    """Get the file name from the absolute path.
+
+    Args:
+        test_path: string. The file's absolute path.
+
+    Returns:
+        String. The file path.
+    """
+    if not test_path:
+        sys.stderr.write(
+            'The test path is empty or null.'
+            'It\'s not possible to diff the test coverage correctly.')
+        sys.exit(1)
+
+    return test_path[test_path.rindex('/') + 1:]
+
+
+def get_lcov_file_tests(file_path):
+    """Get all tests from a lcov file, and filters it to return only:
+    - File path
+    - File total lines
+    - File covered lines
+
+    Args:
+      file_path: string. The path of lcov file.
+
+    Returns:
+      String array. An array with all tests filtered, including only important
+      data for the diff.
+    """
+    with python_utils.open_file(file_path, 'r') as f:
+        tests_array = f.read().split('end_of_record')
+        tests_array_filtered = []
+
+        for test in tests_array:
+            lines = [line for line in test.splitlines() if filter_lines(line)]
+            if len(lines) > 0:
+                tests_array_filtered.append(lines)
+
+        return tests_array_filtered
+
+
+def build_tests_fully_coverage_dict():
+    """Build a dict with only fully covered files from develop branch.
+
+    Returns:
+        Dictionary. A dict containing file path, total lines and covered lines
+        of each tested file.
+
+    Raises:
+      Exception: If DEV_LCOV_FILE_PATH doesn't exist.
+    """
+    coverage_dict = {}
+
+    if os.path.exists(DEV_LCOV_FILE_PATH):
+        tests = get_lcov_file_tests(DEV_LCOV_FILE_PATH)
+        for lines in tests:
+            total_lines = lines[1].split(':')[1]
+            covered_lines = lines[2].split(':')[1]
+
+            if total_lines == covered_lines:
+                test_name = get_test_file_name(lines[0])
+                coverage_dict[test_name] = [
+                    int(total_lines),
+                    int(covered_lines)
+                ]
+
+        return coverage_dict
+    else:
```

Comment:
`else` is not necessary. That being said, this construction is a bit strange. I would do this instead:

```python
if not os.path.exists(DEV_LCOV_FILE_PATH):
    raise Exception(...)

..rest of your code...
```

Reply:
Done